VmAgent.Core/VmConfiguration.cs

```diff
@@ -49,7 +50,10 @@ public class VmConfiguration
         // Typically this wouldn't be needed by VM startup scripts (since it might point to a path that only exists inside a container)
         // but the Watson team found an edge case where the VM needs access to it so we made it available to the VM startup scripts as well.
         public const string SharedContentFolderEnvVariable = "PF_SHARED_CONTENT_FOLDER";
-        
+
+        /// Prefix used for all the Public Ip Addresses environment variables (count, ip addresss, fqdn, routing type)
+        public const string PublicIpAddressesEnvVariablePrefix = "PF_PUBLIC_IP_ADDRESSES";
```

Comment:
not strong feelings, maybe add the _ here so the variables later contain only the actual name? But it's OK not to

Reply:
I tried it, but TBH it doesn't look great once you use it in code with the rest. I would prefer to leave it as it is. But thanks for the suggestion.